docs/contract.md

```diff
@@ -0,0 +1,47 @@
+# Our Contract
+
+We revised our contracts early in 2018 to include more specific language about what's expected from both sides,
+such as clearer milestones and how quickly we should provide feedback on updates to your content.
+This page is a simplified human-readable summary of its terms,
+and is not a substitute for the full agreement.
+
+DataCamp's contract with authors is broken into two parts to simplify future agreements:
+a *master agreement* governing our relationship,
+which you only need to sign once,
+and a *content exhibit* with project-specific details.
+This is a shorter document,
+and you will need to sign one copy for each work you do with us
+(not least because it includes delivery dates).
+
+The contract's key features are:
+
+- You grant us a perpetual non-exclusive license to use the material that you develop for us.
+  The "non-exclusive" part means that you can use the material elsewhere,
+  e.g., in other courses or workshops.
+  However, if you provide it through another online course platform,
+  your revenue share from DataCamp will be reduced.
+
+- It defines a timeline for delivering materials
+  (typically six months for a full course, and less for projects and other kinds of content).
+  We are happy to negotiate extensions if real life gets in the way,
+  but if it appears that your other commitments aren't going to let you complete work in a timely fashion,
+  or if the quality of what you deliver doesn't meet our standards
+  even after we have given you feedback and a chance to make corrections,
+  we may give notice that we are terminating the contract.
+
+- If updates to the material are needed,
+  the contract requires us to contact you and give you first option to make the required changes.
+  If you choose not to do so,
+  we may bring in another instructor to do so.
```

Comment:
"we may bring in another instructor and allocate a reasonable portion of future royalties to them"